python/ray/serve/_private/deployment_scheduler.py

```diff
@@ -39,6 +43,10 @@ class ReplicaSchedulingRequest:
     actor_options: Dict
     actor_init_args: Tuple
     on_scheduled: Callable
+    # Placement group bundles and strategy *for this replica*.
+    # These are optional: by default replicas do not have a placement group.
+    placement_group_bundles: Optional[List[Dict[str, float]]] = None
+    placement_group_strategy: Optional[str] = None
```

Comment:
I think we can create a `PlacementGroupDeploymentSchedulingPolicy` that contains `placement_group_bundles` and `placement_group_strategy` and register this policy during `on_deployment_created`.